app/code/Mykhailok/AskAboutThisProduct/Block/Product/View/Question.php

```diff
@@ -0,0 +1,40 @@
+<?php
+declare(strict_types=1);
+
+namespace Mykhailok\AskAboutThisProduct\Block\Product\View;
+
+class Question extends \Magento\Framework\View\Element\Template
+{
+    private array $layoutProcessors;
+    private string $processedJsLayout = '';
+    private \Magento\Framework\Serialize\SerializerInterface $serializer;
+
+    public function __construct(
+        \Magento\Framework\View\Element\Template\Context $context,
+        \Magento\Framework\Serialize\SerializerInterface $serializer,
+        array $data = [],
+        array $layoutProcessors = []
+    ) {
+        parent::__construct($context, $data);
+        $this->layoutProcessors = $layoutProcessors;
+        $this->serializer = $serializer;
+    }
+
+    /**
+     * @inheritdoc
+     * @noinspection PhpUndefinedVariableInspection
+     */
+    public function getJsLayout(): string
+    {
+        if (empty($this->processedJsLayout) && !empty($this->layoutProcessors)) {
+            /** @var \Magento\Checkout\Block\Checkout\LayoutProcessorInterface $processor */
+            foreach ($this->layoutProcessors as $processor) {
+                $jsLayout = $processor->process($this->jsLayout);
```

Comment:
@mykhailok-dv Error here. What if there are multiple processors? You always pass `$this->jsLayout`, and every processor will get initial value instead of te value changed by other processors.